hw04-Proxy/src/main/java/ru/otus/homework/ProxySample.java

```diff
@@ -0,0 +1,38 @@
+package ru.otus.homework;
+
+import ru.otus.homework.logging.Log;
+import ru.otus.homework.logging.Proxibale;
+import ru.otus.homework.logging.TestLogging;
+import ru.otus.homework.logging.TestLoggingInterface;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Method;
+import java.lang.reflect.Proxy;
+import java.util.ArrayList;
+
+public class ProxySample {
+    static <T extends Proxibale> T createClass(T someInterface) throws ClassNotFoundException {
```

Comment:
```suggestion
    static <T extends Proxibale> T createClass(T obj) throws ClassNotFoundException {
```

Это же объект, а не интерфейс.
И лучше стараться не использовать в названиях "some", "my" и тп.
Название все-таки должно нести смысл.